src/components/playground/PropInput.js

```diff
@@ -22,6 +23,9 @@ export default function PropInput({ prop, path, propName }) {
         <FontWeightInput prop={prop} pathToProp={path} propName={propName} />
       );
       break;
+    case CONFIG_INPUT_TYPES.FONT_SIZE:
```

Comment:
No need to add a specific input type for the font size, in the config we can simply change the `inputType` to a `CONFIG_INPUT_TYPES.SLIDER`